iree/compiler/Codegen/Passes.h

```diff
@@ -369,6 +369,9 @@ std::unique_ptr<OperationPass<func::FuncOp>> createLLVMGPUPipeliningPass(
 std::unique_ptr<OperationPass<func::FuncOp>> createLLVMGPUMultiBuffering(
     unsigned numBuffers = 5);
 
+/// Apply transformation to reduce the number of bank conflicts.
```

Comment:
Could you make this explicit to say reduce bank conflicts for shared memory usage (both in the comment and the pass name).

Reply:
done.